qa/workunits/rbd/rbd_mirror_group.sh

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env bash
+#
+# rbd_mirror_group.sh - test rbd-mirror daemon for snapshot-based group mirroring
+#
+# The script starts two ("local" and "remote") clusters using mstart.sh script,
+# creates a temporary directory, used for cluster configs, daemon logs, admin
+# socket, temporary files, and launches rbd-mirror daemon.
+#
+
+set -ex
+
+MIRROR_POOL_MODE=image
+MIRROR_IMAGE_MODE=snapshot
+
+. $(dirname $0)/rbd_mirror_helpers.sh
+
+setup
+testlog "TEST: create group and test replay"
+start_mirrors ${CLUSTER1}
+group=test-group
+create_group_and_enable_mirror ${CLUSTER2} ${POOL} ${group}
+wait_for_group_replay_started ${CLUSTER1} ${POOL} ${group} 0
+wait_for_group_status_in_pool_dir ${CLUSTER1} ${POOL} ${group} 'up+replaying'
+wait_for_group_present ${CLUSTER1} ${POOL} ${group} 'present'
+if [ -z "${RBD_MIRROR_USE_RBD_MIRROR}" ]; then
+  wait_for_group_status_in_pool_dir ${CLUSTER2} ${POOL} ${group} 'down+unknown'
+fi
+
+testlog "TEST: add image to group and test replay"
```

Comment:
when multiple images are in the group for group related operations, does it perform that operation on each images parallel or sequential?